// Copyright 2025 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//
syntax = "proto2";

package testing_extension_namespace;

import "arolla/proto/testing/test.proto";

message BenchmarkExtension {
  extend testing_namespace.Root {
    optional BenchmarkExtension bench_ext = 154223;
  }
  optional int32 x0 = 10;
  optional int32 x1 = 11;
  optional int32 x2 = 12;
  optional int32 x3 = 13;
  optional int32 x4 = 14;
  optional int32 x5 = 15;
  optional int32 x6 = 16;
  optional int32 x7 = 17;
  optional int32 x8 = 18;
  optional int32 x9 = 19;
}
